### PR TITLE
Zipkin shipping

### PR DESCRIPTION
**Pull request by imnotashrimp**
### New pull request

Two changes here:

1. Add Zipkin instructions
2. Change 'custom code' to 'from your code', with accompanying redirects

### CODE REVIEW

**Comment by imnotashrimp** [comment]
@idohalevi @yyyogev 

Find the Zipkin doc here: https://deploy-preview-178--logz-docs.netlify.com/shipping/log-sources/platform-service/zipkin.html

File: _source/logzio_collections/_log-sources/platform-service/zipkin.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    java -Dloader.path='zipkin-logzio.jar,zipkin-logzio.jar!lib' -Dspring.profiles.active=logzio -cp zipkin.jar org.springframework.boot.loader.PropertiesLauncher
+    ```
+
+    <div class="info-box tip">
+      You can optionally run two Zipkin-Logzio Trace Storage instances if you want to separate shipping and reading of your traces.
+
+      If you do, then the required fields change a bit from what's shown in the Parameters list:
+
+      * The **shipping instance** requires `STORAGE_TYPE=logzio`, `LOGZIO_ACCOUNT_TOKEN`, and `LOGZIO_LISTENER_HOST` (if you're not shipping to the default listener).
+      * The **reading instance** requires `STORAGE_TYPE=logzio`, `LOGZIO_API_TOKEN`, and `LOGZIO_API_HOST`.
```
Comment (yyyogev) [inline _source/logzio_collections/_log-sources/platform-service/zipkin.md]: @imnotashrimp let's keep consistency of the required parameters, either both LOGZIO_API_HOST and LOGZIO_LISTENER_HOST should be required or neither of them..

File: _source/logzio_collections/_log-sources/platform-service/zipkin.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    : {% include log-shipping/replace-vars.html listener=true %}
+      <!-- logzio-inject: listener-url -->
+
+    LOGZIO_API_HOST <span class="default-param">`api.logz.io`</span>
+    : Required to read back spans from Logz.io. \\
+      Replace `<API-URL>` with your region's base API URL.
+      For more information on finding your account's region, see [Account region]({{site.baseurl}}/user-guide/accounts/account-region.html).
+
+    STRICT_TRACE_ID <span class="default-param">`true`</span>
+    : Use `false` if your version of Zipkin server generates 64-bit trace IDs.
```
Comment (yyyogev) [inline _source/logzio_collections/_log-sources/platform-service/zipkin.md]: @imnotashrimp Zipkin 1.15 is the first version that generates 128-bit trace IDs

Comment (imnotashrimp) [reply _source/logzio_collections/_log-sources/platform-service/zipkin.md]: done. will be in next push.

**Comment by imnotashrimp** [comment]
@yyyogev Am I good to merge this doc?

File: _source/logzio_collections/_log-sources/platform-service/zipkin.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+    LOGZIO_LISTENER_HOST <span class="default-param">`listener.logz.io`</span>
+    : {% include log-shipping/replace-vars.html listener=true %}
+      <!-- logzio-inject: listener-url -->
+
+    LOGZIO_API_HOST <span class="default-param">`api.logz.io`</span>
+    : Required to read back spans from Logz.io. \\
+      Replace `<API-URL>` with your region's base API URL.
+      For more information on finding your account's region, see [Account region]({{site.baseurl}}/user-guide/accounts/account-region.html).
+
```
Comment (yyyogev) [inline _source/logzio_collections/_log-sources/platform-service/zipkin.md]: @imnotashrimp please add the 2 new parameters

Comment (imnotashrimp) [reply _source/logzio_collections/_log-sources/platform-service/zipkin.md]: In the latest commit, should show up soon

**Pull request merged**